src/test/java/com/tictactoe/EasyComputerPlayerTest.java

```diff
@@ -0,0 +1,41 @@
+package com.tictactoe;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class EasyComputerPlayerTest {
+
+    private void populateBoard(Board board, int numberOfCells) {
+        for (int i = 0; i < numberOfCells; i++) {
+            if (i % 2 == 0) {
+                board.insertSymbol("X", i);
+            } else {
+                board.insertSymbol("O", i);
+            }
+        }
+    }
+
+    private void populateAnotherBoard(Board board) {
+        board.insertSymbol("O", 0);
+        board.insertSymbol("X", 4);
+    }
+
+    @Test
+    public void easyComputerPlayerTakesFirstAvailableSpotOnBoard_Case1() throws Exception {
+        Board board = new Board();
+        this.populateBoard(board,4);
+        EasyComputerPlayer simpleAI = new EasyComputerPlayer("X", board);
+        int cell = simpleAI.obtainValidCellSelection();
+        assertEquals(5, cell);
+    }
+
+    @Test
+    public void easyComputerPlayerTakesFirstAvailableSpotOnBoard_Case2() throws Exception {
```

Comment:
I added a test for this. When the board is full, the function tell the computer to pick cell 10, which technically doesn't exist. But the game would have been terminated already due to a prior check by `gameInPlay()` in the `Game` class. So the computer would never get to this point.